packages/applications/ssr/src/components/pages/garanties-financières/détails/components/GarantiesFinancièresDépôtEnCours.tsx

```diff
@@ -8,16 +8,19 @@ import { Heading2 } from '@/components/atoms/headings';
 import { CallOut } from '@/components/atoms/CallOut';
 import { formatDateForText } from '@/utils/formatDateForText';
 
+import { ValiderDépôtEnCoursGarantiesFinancières } from '../../dépôt/modifier/valider/validerDépôtEnCoursGarantiesFinancières';
+import { SupprimerDépôtEnCoursGarantiesFinancières } from '../../dépôt/modifier/supprimer/SupprimerDépôtEnCoursGarantiesFinancières';
+
 import { DépôtGarantiesFinancières } from './GarantiesFinancièresHistoriqueDépôts';
 
-type GarantiesFinancièresDépôtEnCoursProps = {
-  dépôt: DépôtGarantiesFinancières & { action?: 'modifier' | 'instruire' };
+export type GarantiesFinancièresDépôtEnCoursProps = {
+  dépôt: DépôtGarantiesFinancières & { actions?: Array<'modifier' | 'instruire' | 'supprimer'> };
```

Comment:
Faut pas que `actions` soit optionnel ici, au pire c'est un array vide

```suggestion
  dépôt: DépôtGarantiesFinancières & { actions: Array<'modifier' | 'instruire' | 'supprimer'> };
```